### DISCUSSION TITLE
AI

### DISCUSSION THREAD

**Discussion by Diae98** [flet-dev/flet#1509] [General]
How I can add  chat bot AI in my app !?

**Comment by SKbarbon** [comment]
What is the issue with flet, its only a UI package

**Comment by Godmemer420** [comment]
You need to you the request packages and also look for the API of the chat bot that you are using, it'll probably cost. Look at some tutorials for the request library and how to access an API to get better understanding of it and then you can get the user input of your users, send it with the API request and then display the response to the user.

**Comment by Diae98** [comment]
thank you guys...
